recipe/conda_build_config.yaml

```
@@ -364,6 +364,8 @@ cairo:
   - 1.16
 cfitsio:
   - 3.470
+cutensor:
+  - 1.2.2.5
```

This should match the `run_exports` version
```suggestion
  - 1
```

---

Hmmm...I don't know if cuTENSOR provides any guarantee, and I'd prefer to follow NCCL's pinning. Sounds like I should fix `pin_subpackage` in cuTENSOR's recipe first?

---

> Sounds like I should fix pin_subpackage in cuTENSOR's recipe first?

Yes, but I would go with the current run_exports as the SONAME only has the major version and I'd rather we trust the upstream developers to bump the SONAME if they make a breaking change.

---

Make sense, thanks @isuruf! Applied your patch.